superset-frontend/src/dashboard/components/SaveModal.tsx

```diff
@@ -137,17 +152,17 @@ class SaveModal extends React.PureComponent {
         t('You must pick a name for the new dashboard'),
       );
     } else {
-      this.onSave(data, dashboardId, saveType).then(resp => {
+      this.onSave(data, dashboardId, saveType).then((resp: any) => {
```

Comment:
I believe a `SupersetResponse` type exists, perhaps in `superset-ui/core`? That would be preferable to `any` here